src/api/hooks/Membership.tsx

```diff
@@ -1,8 +1,39 @@
-import { useQuery } from 'react-query';
+import { useMutation, UseMutationResult, useQuery, useQueryClient } from 'react-query';
 import API from 'api/api';
-import { Membership } from 'types/Types';
+import { Membership, RequestResponse, User } from 'types/Types';
+import { MembershipType } from 'types/Enums';
 
 const QUERY_KEY = 'membership';
```

Comment:
```suggestion
export const MEMBERSHIP_QUERY_KEY = 'membership';
```

Eksporter slik at andre filer kan invalidere membership-cache om nødvendig